src/components/App.js

```diff
@@ -59,12 +83,16 @@ class App extends Component {
     });
   };
 
-  onSetPincode = (pincode = "") => {
-    if (!pincode || pincode.endsWith("--")) {
-      this.onError('Invalid pincode!');
-      return;
-    }
-    this.props.initConnection(pincode);
+  onSetIsAudioEnabled = (isAudioEnabled) => {
+    this.setState({
+      isAudioEnabled: isAudioEnabled
+    });
+    localStorage.setItem("isAudioEnabled", isAudioEnabled);
+  };
+
+  onSetUsername = (username) => {
+    this.props.setUsername(username);
+    this.onToggleModalVisibility('username', false);
```

Comment:
Snuck in on bad merge with develop. Will remove.

Reply:
Feel free to merge this in once you do! Lookin' good :sunglasses: